app/javascript/packages/document-capture/components/suspense-error-boundary.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, Suspense } from 'react';
+import type { ReactNode } from 'react';
+import { RetrySubmissionError } from './submission-complete';
+
+interface SuspenseErrorBoundaryProps {
+  // Fallback to show while suspense pending.
+  fallback: NonNullable<ReactNode>|null;
+  // Error callback.
+  onError: (error: Error)=>void;
+  // Error instance caught to allow for acknowledgment of rerender, in order to prevent infinite rerendering.
```

Comment:
I think we typically use the JSDoc style param documentation
```suggestion
  /**
   * Fallback to show while suspense pending.
   */
  fallback: NonNullable<ReactNode>|null;
  /**
    * Error callback.
    */
  onError: (error: Error)=>void;
  /**
    * Error instance caught to allow for acknowledgment of rerender, in order to prevent infinite rerendering.
    */
```